src/modules/tas_optimizer/optimizer.rs

```diff
@@ -535,7 +535,31 @@ fn mutate_single_frame_bulk<R: Rng>(hltas: &mut HLTAS, rng: &mut R) -> usize {
         } else if p < 0.1 {
             StrafeType::MaxAngle
         } else {
-            StrafeType::MaxAccel
+            // Constant yawspeed will not be selected unless specified in bulk.
+            if let StrafeType::ConstYawspeed(yawspeed) = *type_ {
+                StrafeType::ConstYawspeed(yawspeed)
+            } else {
+                StrafeType::MaxAccel
+            }
+        };
```

Comment:
This will let const yawspeed change to max deccel or max angle, but never to max accel, is that intentional?

Reply:
Oh, definitely unintentional. I can make it more conditional.